src/store/user/asyncActions.js

```diff
@@ -0,0 +1,22 @@
+import {
+  setUserErrorAction,
+  setUserAction,
+  setUserStatusAction,
+} from "./slice";
+import formatRequest from "../../helpers/formatRequest";
+const STATUS_ERROR = "error";
```

Comment:
statuses should be general abd stored in a separated file f.e. `constants/requests.js` or `utils/requests.js`